documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc

```diff
@@ -48,8 +48,10 @@ Each of the annotations supports a set of attributes that you can use to further
 | @Keyword
 | searchable, sortable, projectable, aggregable, indexNullAs, normalizer, norms
 
+ifdef::community[]
```

Comment:
@rchettri16 My understanding is that only 'termVector' should be kept only in the community version